### PR TITLE
doc: sync Security-Team.md GitHub teams membership

### PR DESCRIPTION
**Pull request by richardlau**
Results of running `ncu-team sync Security-Team.md` from
node-core-utils.

### CODE REVIEW

File: Security-Team.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ includes a fix, documentation, an informational CVE or blog post.
 - [@jasnell](https://github.com/jasnell) - James M Snell
 - [@mcollina](https://github.com/mcollina) - Matteo Collina
 - [@MylesBorins](https://github.com/MylesBorins) - Myles Borins
+- [@ronag](https://github.com/ronag) - Robert Nagy
 - [@shigeki](https://github.com/shigeki) - Shigeki Ohtsu
 - [@vdeturckheim](https://github.com/vdeturckheim) - Vladimir de Turckheim
-- [@watson](https://github.com/watson) - Thomas Watson
```
Comment (mcollina) [inline Security-Team.md:46]: @watson has access to h1 and node-private

Comment (richardlau) [reply Security-Team.md:46]: but not in https://github.com/orgs/nodejs/teams/security-triage/members which is what this list syncs with.

Comment (mhdawson) [reply Security-Team.md:46]: I don't think @watson has as much time as he'd hoped to get involved on this front.  @watson should you be added to the  https://github.com/orgs/nodejs/teams/security-triage/members?

Comment (watson) [reply Security-Team.md:46]: Sorry, it was my intend be have been more involved, but I've been overloaded for a while so it hasn't been possible. However, I would still like to be part of the team if possible.

Comment (mhdawson) [reply Security-Team.md:46]: @richardlau I've added @watson to the list you mentioned so you should be able to re-run the sync and get this landed

Comment (richardlau) [reply Security-Team.md:46]: Re-synced. @mcollina PTAL.

**Comment by richardlau** [comment]
We might need to sync again after https://github.com/nodejs/node/pull/34809?

**Comment by mhdawson** [comment]
@richardlau can you resync again and then we'll try to get it landed. @mcollina I assume you can remove your objetion as it was updated?

**Comment by richardlau** [comment]
Resynced.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**